doc/commands.md

```diff
@@ -1280,8 +1280,9 @@ Option | Description | Type | Notes
 `--default.okapi` | Default Okapi URL for CLI config | string | default: "http://localhost:9130"
 `--default.tenant` | Default tenant for CLI config | string | default: "diku"
 `--dir` | Directory to create | string | default: "stripes"
+`--fetch` | Fetch module names from Github | boolean | default: false
```

Comment:
Does this flag work on its own without `--modules`?

Reply:
I don't think so, [based on this](https://github.com/folio-org/stripes-cli/pull/222/files#diff-154ab32bbd9bf2378b7f99413bb29c75R170).

But I honestly don't understand why you need to run it as `--modules --fetch` based on the code, or intuitively. Can you shed some light on this @ryandberger?

Reply:
If you omit `--modules` you immediately get prompted with the outdated list, so it is needed to bypass it. @ttamj's suggestion below to introduce a new command sounds like a better approach to me to make this cleaner and less confusing.

Reply:
@doytch For reference, the `promptMiddleware` attached to the `--modules` option was causing this behavior Ryan describes.  When [included on an option like this](https://github.com/folio-org/stripes-cli/blob/9fab48dcc4e9db9a8666ef4e4d28b710b864626b/lib/commands/workspace.js#L125-L127), the promptMiddleware will be invoked before the command executes causing the CLI to [wait for user input](https://github.com/folio-org/stripes-cli/blob/9fab48dcc4e9db9a8666ef4e4d28b710b864626b/lib/cli/prompt-middleware.js#L23).  Defining a value for the option on the command line (apparently even an empty one) satisfies the promptMiddleware and so it doesn't block.   See also: [Interactive Input](https://github.com/folio-org/stripes-cli/blob/master/doc/dev-guide.md#interactive-input)